apps/judicial-system/backend/src/app/modules/notification/notification.service.ts

```diff
@@ -816,8 +882,14 @@ export class NotificationService {
 
   private async sendModifiedNotifications(
     theCase: Case,
-    user: User,
+    user?: User,
   ): Promise<SendNotificationResponse> {
+    if (!user) {
```

Comment:
Same here, why change the user from being required?

Reply:
Same reason.